src/components/accordion/accordion.tsx

```diff
@@ -32,16 +32,13 @@ const AccordionItemHeader = styled.div`
   cursor: pointer;
   display: flex;
   align-items: center;
-  border-color: rgb(213, 213, 213) currentcolor currentcolor;
+  border-color: #d5d5d5 currentcolor currentcolor;
   border-style: solid none none;
-  color: rgb(45, 45, 45);
+  color: #2d2d2d;
```

Comment:
I'm not convince on default rules about colors.

`rgba(rrr, ggg, bbb, 0.5)` should not transform to `rgb(rrr ggg bbb / 50)`. and `rgb()` should not be transformed to hexadecimal syntax.

Bundling could include this kind of aggressive transformation, but a linter, I'm not sure.

Reply:
I did the hexadecimal syntax manually because of the complex background below that took 3 lines with prettier. This is more compact (and I put everything in hexadecimal in this file for consistency)

Reply:
I don't have strong opinion on the format, but I don't think the goal of the default rules is to do an aggressive transformation for optimization.

This is the error if I put `rgba(r,g,b,0.2)`:

  37:10  ✖  Expected modern color-function notation  color-function-notation
  37:24  ✖  Expected "0.2" to be "20%"               alpha-value-notation

Reply:
Lets trust stylelint recommended config so. We will get used to it.